api/appendices/resource_attributes.md

```diff
@@ -151,7 +151,6 @@ request.
 | per\_time         | string  | Measured per time, **hourly**, **daily**, **weekly**, **monthly** or **yearly**                                                          |
 | per\_unit         | string  | Measured per unit, i.e. **megabytes**, **gigabytes**, etc.                                                                               |
 | chargeback\_rate  | hash    | `id` or `href` from `api/chargebacks` collection.                                                                                        |
-| detail\_measure   | hash    | `id` or `href` from `api/measure` collection. Details of related measure.                                                                |
```

Comment:
`detail\_measure` is on `chargeable_fields`.  Here it is deprecated so I am removing it.